lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -56,7 +56,6 @@
 <article class="course${mode_class}">
   <% course_target = reverse('info', args=[unicode(course_overview.id)]) %>
   <section class="details" aria-labelledby="details-heading-${course_overview.number}">
-      <h2 class="hd hd-2 sr" id="details-heading-${course_overview.number}">${_('Course details')}</h2>
```

Comment:
Nevermind. I see that this is called out specifically in the accessibility guidelines section of the story. Thanks.